_bg-material/glossary.md

```diff
@@ -216,7 +222,10 @@ Please note that this Glossary is **work in progress**. Do you encounter missing
     :   see also **frame**
 
 *	readthrough transcript
```

Comment:
What is the difference between a chimeric transcript and a readthrough transcript? Does chimeric transcript refer to the outcome whereas readthrough implies a genomic origin of a chimeric transcript?

If true, consider a change like this: "a chimeric transcript that results from the transcription of the fragments of two or more genes that have been rearranged relative to typical positions and that are contiguously transcribed."

Reply:
Thanks @reece. All readthrough transcripts are chimeric transcripts. They have the additional properties as described, that the two (or more) genes involved are found on the same chromosomal region, on the same strand, and are typically adjacent to one another.

This distinction is necessary, as HGVS recommends the use of the HGNC gene fusion syntax, which delineates most gene fusions with `::` but differentiates these from readthrough transcripts, which are delineated by `-`.

From what I understand, the intended usage of the readthrough transcript syntax is to enable composite gene symbols from readthrough transcripts that are "natively" (i.e. by reference genome assembly) adjacent to one another as minted "readthrough locus" concepts in CCDS and HGNC gene symbols. For example, the [INS-IGF2 Readthrough Locus](https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:33527), comprised of the [INS](https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:6081) and [IGF2](https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:5466) gene loci.

In order to be compatible with HGNC recommendations, we needed to differentiate between readthrough transcripts and other chimeric transcripts (https://github.com/HGVSnomenclature/HVNC/discussions/10#discussioncomment-4828155). Since HGVS _also_ follows the HGNC recommendations, I felt it was prudent to also include this definition here. Unfortunately, readthrough transcripts can technically also be created by rearranged genes, which (AFAIK) _is not_ the intended usage of the HGNC `-` delineated readthrough syntax. This nuance is described [in this note](https://fusions.cancervariants.org/en/latest/terminology.html#read-through-note) in the VICC gene fusion specification, but I had previously left that out of this PR.

I have added some draft text in 73417f2 to capture this nuance and help guide users to understand why the distinction is necessary. @reece, do you think this addition sufficiently clarifies?

Reply:
Thanks for the explanation. That's more complex than I anticipated. It also raises a new question: [HGNC defines readthrough](https://www.genenames.org/help/symbol-report/) like this:

> readthrough - a naturally occurring transcript containing coding sequence from two or more genes that can also be transcribed individually

Can we (and shouldn't we?) use that definition as-is?

Or, how about this the following?

> readthrough transcript - a naturally occurring chimeric transcript containing coding sequence from two or more genes that can also be transcribed individually, that on the same chromosomal region and strand, and that are typically adjacent to one another.

I'm trying to use HGNC as much as possible while also recognizing the needs for HGVS.

Reply:
I think that the HGNC definition does not match their intended usage. These are the concerns I had with the HGNC definition:

1) Read-through transcripts can involve coding genes, but genes can also be non-coding; are non-coding read-throughs (transcribed by the same machinery / mechanism as coding read-throughs) not read-throughs by virtue of not having coding sequence? I think this is a narrow and imprecise view of readthrough transcription.
2) What is a naturally occurring transcript? If a person has a germline rearrangement relative to a reference, which produces a novel transcript by a read-through mechanism, is that natural? If a cancer cell has a somatic rearrangement that does the same, is that natural? If we are to use this definition, I think we will also need to define what mean by naturally occurring, which seems burdensome.

I found the [Ensembl definition](https://www.ensembl.info/2019/02/11/annotating-readthrough-transcription-in-ensembl/#:~:text=Readthrough%20transcripts%20are%20RNA%20molecules,typically%20adjacent%20to%20one%20another.) to be more precise than the HGNC definition, which is what I based the submitted definition on. However, I agree we should try to align with the HGNC definition, so I propose striking my previous addition and instead using the following (new text bolded):

> a chimeric transcript where the two (or more) genes involved **can also be transcribed individually,** and are found on the same chromosomal region, on the same strand, and typically adjacent to one another.

@reece what do you think of this revised text?